olp-cpp-sdk-authentication/include/olp/authentication/TokenProvider.h

```diff
@@ -39,7 +41,8 @@ namespace authentication {
  * @param minimumValidity the minimum token validity to use in seconds.
  */
 template <long long MinimumValidity>
-class TokenProvider {
+class OLP_SDK_DEPRECATED("Deprecated. Will be removed by 04.2020")
+TokenProvider {
```

Comment:
It seems that this line is not formatted, is it?

Reply:
It is formatted with clang-format-6.0